src/guides/thingsboard.md

```diff
@@ -1,10 +1,10 @@
 # ThingsBoard getting started
 
-[ThingsBoard](https://www.thingsboard.io) is an Open-source IoT Platform for
+[ThingsBoard](https://www.thingsboard.io) is an Open-core IoT Platform for
```

Comment:
I'm not sure how relevant this is, the ThingsBoard website calls it open-source.